README.md

```diff
@@ -113,7 +113,11 @@ Left would be created with `nnoremap <silent> <c-h> :TmuxNavigateLeft<cr>`.
 
     let g:tmux_navigator_save_on_switch = 1
 
-This will execute the update command on leaving vim to a tmux pane. Default is Zero
+This will execute the `update` command on leaving vim to a tmux pane. Default is Zero
+
+    let g:tmux_navigator_save_all_buffers_on_switch = 1
```

Comment:
I like @blueyed's idea of opting into the existing variable. What do you think of the following for the help text:

> You can configure the plugin to write the current buffer, or all buffers, when navigating from Vim to tmux. This functionality is exposed via the `g:tmux_navigator_save_on_switch` variable, which can have either of the following values:

> | Value | Behavior |
> | ----- | --------- |
> | 1 | `:update` (write the current buffer, but only if changed) |
> | 2 | `:wall` (write all buffers) |
>
> To configure, add the following (with the desired value) to your `~/.vimrc`:
> 
> ``` vim
> " Write all buffers before navigating from Vim to tmux pane
> let g:tmux_navigator_save_on_switch = 2
>  ```